photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -642,12 +652,13 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     }
     val downSamplerFactory = DownSamplerHelper.buildFactory(task)
     val lockedCoordinates = get(partialRetrainLockedCoordinates).getOrElse(Set())
+    val indices = get(interceptIndices).getOrElse(Seq.empty[Int])
 
     // Create the optimization coordinates for each component model
     val coordinates: Map[CoordinateId, C forSome { type C <: Coordinate[_] }] =
-      updateSequence
-        .map { coordinateId =>
-          val coordinate: C forSome { type C <: Coordinate[_] } = if (lockedCoordinates.contains(coordinateId)) {
+      updateSequence.zip(indices)
+        .map { case (coordinateId, interceptIndex) => {
+          val coordinate: C forSome {type C <: Coordinate[_]} = if (lockedCoordinates.contains(coordinateId)) {
```

Comment:
Add the white spaces back.

Reply:
Ack.

Reply:
See above comments. Don't `zip` with `Seq` - change parameter type and access it by coordinate ID like the datasets and configurations below.